packages/fxa-payments-server/src/lib/errors.ts

```diff
@@ -163,10 +163,37 @@ function getErrorMessage(error: undefined | StripeError | GeneralError) {
   return errorToErrorMessageMap[lookup];
 }
 
+// fallback error messages
+enum getDefaultErrorMessage {
+  // specific error messages
+  'expired-card-error' = 'It looks like your credit card has expired. Try another card.' as any,
+  'insufficient-funds-error' = 'It looks like your card has insufficient funds. Try another card.' as any,
+  'withdrawal-count-limit-exceeded-error' = 'It looks like this transaction will put you over your credit limit. Try another card.' as any,
+  'charge-exceeds-source-limit' = 'It looks like this transaction will put you over your daily credit limit. Try another card or in 24 hours.' as any,
+  'instant-payouts-unsupported' = 'It looks like your debit card isn’t setup for instant payments. Try another debit or credit card.' as any,
+  'duplicate-transaction' = 'Hmm. Looks like an identical transaction was just sent. Check your payment history.' as any,
+  'coupon-expired' = 'It looks like that promo code has expired.' as any,
+  'country-currency-mismatch' = 'The currency of this subscription is not valid for the country associated with your payment.' as any,
+  'currency-currency-mismatch' = 'Sorry. You can’t switch between currencies.' as any,
+  'no-subscription-change' = 'Sorry. You can’t change your subscription plan.' as any,
+  'iap-already-subscribed' = 'You’re already subscribed through the { $mobileAppStore }.' as any,
+
+  // generic messages for groups of similar errors
+  'card-error' = 'Your transaction could not be processed. Please verify your credit card information and try again.' as any,
+  'basic-error-message' = 'Something went wrong. Please try again later.' as any,
+  'payment-error-1' = 'Hmm. There was a problem authorizing your payment. Try again or get in touch with your card issuer.' as any,
+  'payment-error-2' = 'Hmm. There was a problem authorizing your payment. Get in touch with your card issuer.' as any,
+  'payment-error-3b' = 'An unexpected error has occurred while processing your payment, please try again.' as any,
+  'fxa-account-signup-error-2' = 'A system error caused your { $productName } sign-up to fail. Your payment method has not been charged. Please try again.' as any,
+  'newsletter-signup-error' = 'You’re not signed up for product update emails. You can try again in your account settings.' as any,
+  'fxa-post-passwordless-sub-error' = 'Subscription confirmed, but the confirmation page failed to load. Please check your email to set up your account.' as any,
+}
```

Comment:
Okay thanks; yeah that file (`errors.ts`) needs some TLC/refactoring for sure. Why is `getErrorMessage` returning the l10n id instead of the error message? It should be renamed to `getErrorMessageId` or something.